### DISCUSSION TITLE
Improve Money property type precision to give ability to store cryptocurrency values

### DISCUSSION THREAD

**Discussion by joeberetta** [mnfst/llm-gateway#230] [Feature request]
As said in the [docs](https://manifest.build/docs/properties#money)
>  A money field with a currency. Money properties can have up to 2 digits after coma.

While it’s ok w/ fiat currencies (eur/usd/etc) is not the best choice for crypto monetary system
Cuz e.g. Bitcoin (BTC) has up to 8 digits after comma

**Comment by brunobuddy** [comment]
@joeberetta sounds fun !

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
